src/xenia/kernel/xfile.h

```diff
@@ -76,6 +76,7 @@ class X_FILE_DIRECTORY_INFORMATION {
 class XFile : public XObject {
  public:
   static const XObject::Type kObjectType = XObject::Type::File;
+  static constexpr uint16_t kMaximalPathLength = 260;
```

Comment:
"Maximal" is an uncommon word in coding, usually it's "maximum" or (like we usually use) just "max". Also a comment saying that it includes the null terminator would be good, or even changing this to 259 because length, unlike buffer size, usually doesn't include anything but the character count itself.

Reply:
For lengths, size_t is a more suitable type also.

Reply:
Done